stoservice/src/main/java/com/tutrit/stoservice/bean/Car.java

```diff
@@ -10,12 +10,12 @@ public class Car {
     private String generation;
     private String modification;
     private String engine;
-    private int year;
+    private String year;
```

Comment:
Why did you change the type of the variable?

Reply:
I wouldn't change